src/bundle/Resources/views/content/tabs/versions/table.html.twig

```diff
@@ -32,6 +41,15 @@
             <td>
                 {{ version.modificationDate|localizeddate('medium', 'short', app.request.locale) }}
             </td>
+            {% if isDraft is defined %}
+                <td>
+                    <a href="#" class="table-iconbutton">
```

Comment:
I assumed some CSS would be added at a later point for these to be displayed and it would be fine to leave like this for now but I'll update to your suggested value